includes/Core/Modules/Module.php

```diff
@@ -155,6 +155,15 @@ public function __construct(
 	 */
 	abstract public function register();
 
+	/**
+	 * Gets Module public name.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string Formatted module name.
+	 */
+	abstract public function get_public_name();
```

Comment:
Why was this added? The name is already publicly accessible along with other data from `info` dynamically https://github.com/google/site-kit-wp/blob/d952edb61a7604dede836bc3df8c8b8eddc2afbd/includes/Core/Modules/Module.php#L42

Reply:
Ugh... you are right, my bad, I forgot we have it already. So, we can probably replace that `get_public_name` method with just `$module->name` calls.